addons/l10n_se/data/account.account.template.csv

```diff
@@ -220,27 +214,24 @@
 "a5410","5410","Förbrukningsinventarier","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5420","5420","Programvaror","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5460","5460","Förbrukningsmaterial","account.data_account_type_expenses","l10nse_chart_template","False"
-"a5480","5480","Arbetskläder","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5500","5500","Reparation och underhåll (gruppkonto)","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5600","5600","Kostnader för transportmedel (gruppkonto)","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5611","5611","Drivmedel för personbilar","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5612","5612","Försäkring och skatt för personbilar ","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5613","5613","Reparation och underhåll av personbilar","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5615","5615","Leasing av personbilar","account.data_account_type_expenses","l10nse_chart_template","False"
-"a5616","5616","Trängselskatt","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5700","5700","Frakter och transporter (gruppkonto)","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5800","5800","Resekostnader (gruppkonto)","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5810","5810","Biljetter","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5820","5820","Hyrbilskostnader","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5831","5831","Kost och logi i Sverige","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5832","5832","Kost och logi i utlandet","account.data_account_type_expenses","l10nse_chart_template","False"
-"a5900","5900","Reklam och PR","account.data_account_type_expenses","l10nse_chart_template","False"
+"a5900","5900","Reklam och PR (gruppkonto)","account.data_account_type_expenses","l10nse_chart_template","False"
 "a6071","6071","Representation, avdragsgill","account.data_account_type_expenses","l10nse_chart_template","False"
 "a6072","6072","Representation, ej avdragsgill","account.data_account_type_expenses","l10nse_chart_template","False"
 "a6090","6090","Övriga försäljningskostnader ","account.data_account_type_expenses","l10nse_chart_template","False"
 "a6100","6100","Kontorsmateriel och trycksaker (gruppkonto)","account.data_account_type_expenses","l10nse_chart_template","False"
 "a6210","6210","Telekommunikation","account.data_account_type_expenses","l10nse_chart_template","False"
-"a6230","6230","Datakommunikation","account.data_account_type_expenses","l10nse_chart_template","False"
```

Comment:
This is not listed as the recommended minimum. However, it's included in the complete BAS 2020 and should not be removed.

Reply:
Not in minimum ( https://github.com/odoo/odoo/pull/53092#discussion_r454259845 )